src/plugins/config/TraceConfig.ts

```diff
@@ -42,7 +46,7 @@ class TraceConfig extends BasePluginConfig {
                     ThundraLogger.getInstance().error(`Cannot parse trace def ${key}`);
                 }
             }
-            if (key.startsWith(envVariableKeys.THUNDRA_LAMBDA_TRACE_INTEGRATIONS)) {
+            if (key.startsWith(envVariableKeys.THUNDRA_LAMBDA_TRACE_INTEGRATIONS_DISABLE)) {
```

Comment:
Adding integration even though it is disabled?

Reply:
seems that it is used for disabled integrations, right?

Reply:
yes, I changed the environment variable. It is used for disabling integration.